peps/pep-0426.rst

```diff
@@ -10,14 +8,15 @@ Discussions-To: distutils-sig@python.org
 Status: Withdrawn
 Type: Informational
 Topic: Packaging
-Content-Type: text/x-rst
 Requires: 440, 508, 518
 Created: 30-Aug-2012
 Post-History: 14-Nov-2012, 05-Feb-2013, 07-Feb-2013, 09-Feb-2013,
               27-May-2013, 20-Jun-2013, 23-Jun-2013, 14-Jul-2013,
               21-Dec-2013
 Replaces: 345
 
+.. withdrawn::
```

Comment:
This PEP already has a relatively short withdrawl section right below, so like you've done elsewhere for those of similar length, I suggest moving it here:

```suggestion
.. withdrawn::
   The ground-up metadata redesign proposed in this PEP has been withdrawn in
   favour of the more modest proposal in :pep:`566`, which retains the basic
   Key:Value format of previous metadata versions, but also defines a standardised
   mechanism for translating that format to nested JSON-compatible data structures.
   
   Some of the ideas in this PEP (or the related :pep:`459`) may still be considered
   as part of later proposals, but they will be handled in a more incremental
   fashion, rather than as a single large proposed change with no feasible
   migration plan.
```

Otherwise, if its really too long, you _could_ link it via a ref target label.